test/unit/specs/methods.spec.js

```diff
@@ -1,14 +1,45 @@
 const AppleSignin = require('ti.applesignin');
+var signInButton = AppleSignin.createLoginButton({
```

Comment:
Use `const` or `let` and avoid `var`

```suggestion
const signInButton = AppleSignin.createLoginButton({
```